src/status_im/contexts/keycard/change_pin/events.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im.contexts.keycard.change-pin.events
+  (:require [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(rf/reg-event-fx :keycard/change-pin.enter-current-pin
+ (fn [_]
+   {:fx [[:dispatch [:navigate-back]]
```

Comment:
I think it would be better to use the `:db` effect and clean-up the pin when the flow starts here instead of having to keep track of all the closing points in the flow to dispatch `:keycard/change-pin.cleanup`. The end result should be more robust and the code in the view cleaner because the view wouldn't be concerned with keeping the state correct via special clean up events. Do you see any downside @Parveshdhull?